lib/tests/streamlit/state/session_state_test.py

```diff
@@ -280,6 +283,7 @@ def test_date_input_serde(self):
         )
         check_roundtrip("date_interval", date_interval)
 
+    @pytest.mark.skip
```

Comment:
Is this test in the process of being revived? Or is there something particularly difficult to work with in the new world that'll force us to revisit the file_uploader serde test later